CHANGELOG.md

```diff
@@ -1,12 +1,20 @@
 Changes from v1.5 to v2.0
 =========================
 
+Dependency Changes
+------------------
+
+- Added LSSTDESC.Coord. (#809b)
+
+
 API Changes
 -----------
 
-
-Dependency Changes
-------------------
+- There were some minor API changes to the Angle and CelestialCoord classes we made when we
+  moved it over into LSSTDESC.Coord.  Some were to sever (weak) ties to other GalSim classes and
+  some were just deemed API improvements.  Probably the most relevant of these for most
+  users will be that Angle.rad is now a property rather than a function.  So the value of an
+  angle in radians is just `theta.rad` not `theta.rad()`. (#809b)
```

Comment:
Major versions are where API changes are allowed.  We'd like to keep them to a minimum of course, but if there are any API improvements that we want to make, this is the time to do it.

Personally, I much prefer this API , making rad a property.  It feels much more Pythonic to me, and I think it was a mistake to have made it a function originally.  (Stemming from my lack of experience in Python at the time I originally wrote that class.)